### ISSUE TITLE
[IC2 integration] Stalling Tick caused by IC2 Energy Grid

### ISSUE THREAD

**Issue by Alexthehuman2** [mekanism/Mekanism#5569]
#### Issue description:
Watchdog kills the server due to a massive tick stall caused by the interaction between ic2's energy grid and Mekanism's. Happens randomly and it kicks the TPS severely until the watchdog kills it. This is not an IC2 problem which was mentioned on [this Reddit Thread](https://www.reddit.com/r/feedthebeast/comments/6qwqq3/i_cant_get_ic2_generator_to_charge_mekanism/) , that IC2 has it's own Energy Grid.

#### Steps to reproduce:
1. Connect some Universal cables into an IC2 Machine or Power Storage
2. Server starts to randomly hang over time with increasing time taken per tick. Plugins still function, yet no interaction between the server and the client.

Happens way too randomly to be able to reproduce reliably, crash log provided should help, it happens prominently after a server restart but there's no guarantee it'll happen.

Potential issues could be:
- Unloaded Mekanism Cables which were mentioned by a few other issues in the past, connecting to IC2 Energy Grid.
- Just a general interaction issue between the two energy networks.

#### Version (make sure you are on the latest version before reporting):
**Forge: forge-14.23.5.2838 (Custom Enigmatica 2 Expert)** 
**Mekanism: Mekanism-1.12.2-9.7.8.379.jar** 
**Other relevant version: SpongeForge-14.23.5.2838, MekanismGenerators-1.12.2-9.7.8.379.jar** 

#### If a (crash)log is relevant for this issue, link it here: (_It's almost always relevant_)

Pastebin link: https://pastebin.com/zU1d37f4

**Comment by thiakil** [comment]
That's not enough information to go on - the watchdog stops it there, but it only means the *whole* tick took that long; it could be anything.

You need to get some profiler data from VisualVM (best) or Sampler. (I *really* need to get around to writing some docs for this).

Regarding the reddit thread, while I'm not sure why it's relevant, assertions that we don't "properly" use IC2's Enet is largely incorrect - we most certainly do register with it.

**Comment by Alexthehuman2** [comment]
Unfortunately I was only an assistant to the server I'm playing on that have this constant crash, and every time the tps starts rising and the watch dog kills it to reboot the server, it always bring up the same lines where the watchdog sees a long tick.

I'll relay this back to the admin to see if we could get some VM profiler, about the IC2, it's because we have scoured over the web and we still couldn't find a middleground, no solution about the energy grid, which we took guess from the crash log produced from every stall.
 
Either way, I'll need to talk to the admins about this, so just bare with me.
